scm_repo_link.go

```diff
@@ -63,6 +65,7 @@ type SCMRepoLink struct {
 	HoursBeforeDeleteOnClose uint              `json:"hours_before_delete_on_close"`
 	HoursBeforeDeleteStale   uint              `json:"hours_before_delete_stale"`
 	LastAutoDeployAt         time.Time         `json:"last_auto_deploy_at"`
+	ForksAllowed             bool              `json:"forks_allowed"`
```

Comment:
Rename as `AutomaticCreationFromForksAllowed`

Reply:
Why hasn't that one the `omitempty` attribute?

Reply:
Omitempty doesn't serialize the field if it doesn't contain any value.

As you see, previous Params objects contains pointers on values, which can be true, false, or nil. The value has two informations : 
- If it has a value or not
- If it has one, the value's content

If there are no value : the field won't be serialized. Because these are params : all values are not mandatory.

But here SCMRepoLink represents a state. It shall have every information, and must not be omitted.